gui/src/components/History.js

```diff
@@ -0,0 +1,47 @@
+import {useEffect, useState} from "react";
+import Button from "react-bootstrap/Button";
+import Accordion from "react-bootstrap/Accordion";
+import Table from "react-bootstrap/Table";
+import Container from "react-bootstrap/Container";
+
+function History({socket, historyMessage}) {
+  const [history, setHistory] = useState([]);
+
+  useEffect(() => setHistory(historyMessage["results"]), [historyMessage]);
+
+  const updateHistory = (event) => {
+    event.preventDefault();
+    const msg = `{
+      "messageType": "REQ-HISTORY"
+    }`;
+    console.log(`Socket state: ${socket.readyState}`);
+    if (socket.readyState === 1) socket.send(msg);
+  };
+
+  return (
+    <Container fluid className="p-5 d-grid gap-4">
+      <Button onClick={updateHistory}>Update History</Button>
+      <Accordion>
+        {history === undefined ? <></> : history.map((suite, index) => (
+          <Accordion.Item eventKey={index} key={index}>
+            <Accordion.Header>Suite {suite["suiteID"]} | {suite["url"]}</Accordion.Header>
```

Comment:
I think calling them Suite N in the UI is kinda confusing but maybe that just me

Reply:
Fixed